conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -936,6 +936,13 @@ msgid "Unable to send email activation link. Please try again later."
 msgstr ""
 
 #: common/djangoapps/third_party_auth/models.py
+msgid ""
+"If this option is not selected, users will not be presented with the provider "
+"as an option to authenticate with on the login screen, but manual "
+"authentication using the correct link is still possible."
+msgstr ""
+
+#: common/djangoapps/third_party_auth/models.py
```

Comment:
I'm not positive either; I know we've been including them on XBlock PRs. Certainly no harm in them being there.